Neos.Flow.Log/Classes/DefaultLogger.php

```diff
@@ -20,6 +21,27 @@
  */
 class DefaultLogger implements LoggerInterface
 {
+    const LOGLEVEL_MAPPING = [
+        LOG_EMERG => LogLevel::EMERGENCY,
+        LOG_DEBUG => LogLevel::DEBUG,
+        LOG_INFO => LogLevel::INFO,
+        LOG_NOTICE => LogLevel::NOTICE,
+        LOG_WARNING => LogLevel::WARNING,
+        LOG_ERR => LogLevel::ERROR,
+        LOG_CRIT => LogLevel::CRITICAL,
+        LOG_ALERT => LogLevel::ALERT
+    ];
+
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    protected $psrLogger;
```

Comment:
Applicable to every place this name is used: `psrLogger` or `PsrLogger` is a really bad name IMHO. It's a logger according to PSR-3, and it does not log PSRs, so it should be `psrThreeCompatibleLogger`, but that is rather long.

Not a blocker for me, but still a tad bit weird.

Reply:
Yeah, happy to have a better name, but as you said, the correct name is a bit unwieldy. Not that that should stop us... Idk, I can change that.